docs/src/pages/components/data-grid/filtering/CustomRatingOperator.js

```diff
@@ -72,7 +88,7 @@ export default function CustomRatingOperator() {
   const { data } = useDemoData({ dataSet: 'Employee', rowLength: 100 });
   const columns = [...data.columns];
   const [filterModel, setFilterModel] = React.useState({
-    items: [{ columnField: 'rating', value: '3.5', operatorValue: 'from' }],
+    items: [{ id: 1, columnField: 'rating', value: '3.5', operatorValue: 'from' }],
```

Comment:
Without the id, the first value update create a rerender, and the lost of the focus in the demo